components/xsd-fu/python/ome/modeltools/property.py

```diff
@@ -345,6 +351,50 @@ def _get_unitsType(self):
         _get_unitsType,
         doc="""The property's units type.""")
 
+    def _get_enumProperties(self):
+        if self.isEnumeration:
+            if self.enumProps is None:
+                self.enumProps = dict()
+
+                try:
+                    root = self.model.schemas['ome']
+                    enum = root.find("{http://www.w3.org/2001/XMLSchema}simpleType[@name='%s']" % self.langType)
```

Comment:
Note that using an xpath expression requires ElementTree 1.3+ (see http://stackoverflow.com/a/4809350). However, our Centos6-based CI bundles an older version of the library which explains the [failures of the build](https://ci.openmicroscopy.org/view/Failing/job/BIOFORMATS-DEV-breaking-build).